src/waveform/waveformwidgetfactory.cpp

```diff
@@ -1168,14 +1171,17 @@ QString WaveformWidgetFactory::buildWidgetDisplayName() const {
         extras.push_back(tr("legacy"));
     }
     if (isOpenGlAvailable() || isOpenGlesAvailable()) {
-        if (WaveformT::useOpenGLShaders()) {
+        if (WaveformT::useOpenGles()) {
+            if (WaveformT::useOpenGLShaders()) {
+                extras.push_back(QStringLiteral("GLSL ES"));
+            } else {
+                extras.push_back(QStringLiteral("GLES"));
```

Comment:
```suggestion
                extras.push_back(QStringLiteral("GL ES"));
```

Reply:
This is currently not visible anyway but this is taken from: [OpenGL ES or GLES](https://en.wikipedia.org/wiki/OpenGL_ES)